superdesk/io/feed_parsers/ap_anpa.py

```diff
@@ -117,7 +117,7 @@ def ap_derive_dateline(self, item):
             html = item.get('body_html')
             if html:
                 parsed = parse_html(html, content='html')
-                for par in parsed.xpath('/html/div/child::*'):
+                for par in parsed.xpath('/html/body/child::*'):
```

Comment:
PR was about mapping only but unit test failed. 
In version 1.7 xpath needs to `/html/body` but in version 1.8 it needs to be `/html/div`. It looks like changes were done after `root element` changes in version 1.8. (https://github.com/superdesk/superdesk-core/blob/master/superdesk/etree.py#L121)